app/models/tweet.rb

```diff
@@ -4,14 +4,7 @@ class Tweet < ActiveRecord::Base
 
   before_validation :add_index, on: :create
 
-  # tweet is 140 char max and tweeter name is 15 char max. so we we let 124 to be the max length so that
```

Comment:
I think these two lines of comments serve as a useful explanation to what otherwise seems like an arbitrary number. If this is still the reason for the character limit, I'd keep the comments in.

Reply:
I definitely see your point.  I'll put this comment back in.